test/test_baza.rb

```diff
@@ -104,29 +106,47 @@ def test_push_with_big_meta
           ]
         )
       end
-    assert(req.join.include?('X-Zerocracy-Meta: '))
+    assert(req['x-zerocracy-meta'])
+  end
+
+  def test_push_compressed_content
+    req =
+      with_http_server(200, 'yes') do |baza|
+        baza.push('simple', 'hello, world!', %w[meta1 meta2 meta3])
+      end
```

Comment:
@tank-bohr Maybe it's better to write like this?
 ```rb
req = with_http_server(200, 'yes') do |baza|
  baza.push('simple', 'hello, world!', %w[meta1 meta2 meta3])
end
```

Reply:
it was [before](https://github.com/yegor256/judges/commit/f5f1d1ef6564be9e80d0fd5ece354dfbc37819da). but not now